ci-operator/step-registry/upi/install/heterogeneous/libvirt/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
```

Comment:
You should be able to just use the group for multiarch approvers, `- multiarch-approvers`, instead of specifying a list of users.